docs/src/pages/quasar-cli/developing-ssr/deploying.md

```diff
@@ -32,3 +32,70 @@ After installing PM2 on your server, your npm start script can look like this in
   "start": "pm2 start index.js"
 }
 ```
+
+## Production scenario
```

Comment:
I'd rename as "Usage with Nginx" instead: there may be many "production scenarios"